source/common/upstream/eds.cc

```diff
@@ -166,5 +166,23 @@ void EdsClusterImpl::onConfigUpdateFailed(const EnvoyException* e) {
   onPreInitComplete();
 }
 
+ClusterImplBaseSharedPtr EdsClusterFactory::createClusterImpl(
+    const envoy::api::v2::Cluster& cluster, ClusterFactoryContext& context,
+    Server::Configuration::TransportSocketFactoryContext& socket_factory_context,
+    Stats::ScopePtr&& stats_scope) {
+  if (!cluster.has_eds_cluster_config()) {
+    throw EnvoyException("cannot create an EDS cluster without an EDS config");
+  }
+
+  // We map SDS to EDS, since EDS provides backwards compatibility with SDS.
```

Comment:
Is this comment relevant here? I don't see any mapping?

Reply:
Let me move the comment